docs/news.rst

```diff
@@ -288,6 +288,14 @@ Backward-incompatible changes
     :class:`~scrapy.http.Request` objects instead of arbitrary Python data
     structures.
 
+*   An additional `crawler` paramter has been added to the `__init__` method of
+    the `scrapy.core.scheduler.Scheduler` class. Custom scheduler subclassses 
```

Comment:
I think this is still not 100% accurate. What do you think of something like the following?
> Custom scheduler subclasses which don't accept arbitrary parameters in their `__init__` method might break because of this change.

Reply:
@elacuesta , you're right. Your sentence is clear & concise.

Reply:
I apologize if I wasn't clear, I didn't mean you should suppress the first sentence (_"An additional `crawler` parameter has been added to the `__init__` method of the `scrapy.core.scheduler.Scheduler` class"_). That sentence is the explanation for the following one.
Also, I just realized that two accent characters are needed for Sphinx to render text as monospace, one means italic. You could check this by running `make html` from the `docs` directory.
Lastly, I think it would be good to refer to the Scheduler class using the `class` directive. There is no API reference page for the Scheduler at the moment, but doing so would make this part link to it if we add it in the future. What I mean is:
```
:class:`scrapy.core.scheduler.Scheduler`
```
Sorry for the back and forth, thanks for your patience.

Reply:
@elacuesta, I'm here to learn. Thanks for taking the time out to help a beginner like me! 
I'll make the requested changes!